src/org/zaproxy/zap/extension/pscanrulesAlpha/XPoweredByHeaderInfoLeakScanner.java

```diff
@@ -36,7 +37,7 @@
  * https://github.com/zaproxy/zaproxy/issues/1169
  * @author kingthorin+owaspzap@gmail.com
  */
-public class XPoweredByHeaderInfoLeakScanner extends PluginPassiveScanner{
+public class XPoweredByHeaderInfoLeakScanner extends PluginPassiveScanner {

```

Comment:
The formatting change could be reverted. [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines